content/articles/geospatial_standards/index.Rmd

```diff
@@ -0,0 +1,90 @@
+---
+title: "Meet some popular open geospatial standards!"
+description: "A short introduction to the powerful GeoPackage, GeoJSON and GeoTIFF standards"
+author: "Floris Vanderhaeghe"
+date: 2019-11-19
+csl: ../inbo.csl
+bibliography: ../reproducible_research.bib
+categories: ["r", "gis"]
+tags: ["gis", "r", "open science"]
+output: 
+    md_document:
+        preserve_yaml: true
+        variant: gfm
+---
+
+Some inspiration for this post came from the beautiful books of @lovelace_geocomputation_2019, @pebesma_edzer_spatial_2019 and @heijmans_spatial_2019, and from various websites.
+
+## Why use open standards?
+
+- Open file standards ease collaboration, portability and compatibility between users, machines and applications.
+- Their (file) structure is fully documented.
+  - Consequently, scientists and programmers can build new software / packages and make innovations that use these standards, while maintaining interoperability with existing applications.
+  - And, it guarantees that your data will still be readable in a hundred years from now, independently of which IT corporation or software is dominant at that time...
```

Comment:
Well, at least the format is fully specified, so at least in theory I think it will always be possible to read such data. Not sure though, whether in this sense caveats still exist with binary data. You will indeed need (make/adapt) software that is capable of reading a 100-year old version of a standard, but the same holds for text data.

@peterdesmet what is your view on this? Should the statement be nuanced?

The OGC [states](https://www.opengeospatial.org/standards) this:

> These standards are the main "products" of the Open Geospatial Consortium and have been developed by the membership to address specific interoperability challenges. Ideally, when OGC standards are implemented in products or online services by two different software engineers working independently, the resulting components plug and play, that is, they work together without further debugging.

Reply:
A while ago, I met a [post](https://www.linuxjournal.com/content/open-source-eternal) on this topic, which wants to confirm the general applicability of open standards still being readable far in the future:

>The only hope of reading ancient file formats is if they are completely open. That way, readers and file conversion tools can be built with relative ease. Similarly, any programs that are created for projects looking at very long-term preservation of digital material must be open source, so that they too can be examined in detail, modified and built upon. Even after just 25 years, we know that free software has won. But it is also highly likely that its success will be very long term, assuming human culture survives with any continuity. Open source—and only open source—is eternal.

Reply:
Would rephrase to:

> And, there is a much higher chance that your data will still be readable in the future.

Reply:
Thanks for the suggestion, will write something in that sense.

Reply:
Done in de81b86